scripts/files/fs_s3.py

```diff
@@ -168,15 +167,16 @@ def prefix_from_path(path: str) -> str:
     return path.replace(f"s3://{bucket_name}/", "")
 
 
-def list_json_in_uri(uri: str, s3_client: Optional[client]) -> List[str]:
-    """Get the `JSON` files from a s3 path
+def list_files_in_uri(uri: str, suffixes: Tuple[str, ...], s3_client: Optional[client]) -> List[str]:
```

Comment:
Since `suffixes` is arbitrary length, while n-tuples are fixed length, it might be more appropriate to use either `List[str]` or even `Iterable[str]`.